composer.json

```diff
@@ -16,52 +16,62 @@
         "source": "https://github.com/zfcampus/zf-apigility",
         "issues": "https://github.com/zfcampus/zf-apigility/issues"
     },
-    "repositories": [
-        { "type": "composer", "url": "https://packages.zendframework.com/" }
-    ],
     "extra": {
         "branch-alias": {
             "dev-master": "1.2-dev",
             "dev-develop": "1.3-dev"
+        },
+        "zf": {
+            "module": "ZF\\Apigility"
         }
     },
     "require": {
-        "php": ">=5.5",
-        "rwoverdijk/assetmanager": "~1.3",
-        "zendframework/zend-db": "~2.3",
-        "zendframework/zend-eventmanager": "~2.3",
-        "zendframework/zend-mvc": "~2.3",
-        "zendframework/zend-paginator": "~2.3",
-        "zendframework/zend-servicemanager": "~2.3",
-        "zendframework/zend-stdlib": "~2.3",
-        "zfcampus/zf-api-problem": "~1.0",
-        "zfcampus/zf-apigility-provider": "~1.0",
-        "zfcampus/zf-content-negotiation": "~1.0",
-        "zfcampus/zf-content-validation": "~1.0",
-        "zfcampus/zf-hal": "~1.0",
-        "zfcampus/zf-mvc-auth": "~1.0",
-        "zfcampus/zf-oauth2": "~1.0",
-        "zfcampus/zf-rest": "~1.0",
-        "zfcampus/zf-rpc": "~1.0",
-        "zfcampus/zf-versioning": "~1.0"
+        "php": "^5.6 || ^7.0",
+        "zendframework/zend-db": "^2.8.1",
+        "zendframework/zend-eventmanager": "^2.6.3 || ^3.0.1",
+        "zendframework/zend-loader": "^2.5.1",
```

Comment:
It's required by `ZF\Apigility\Autoloader`, which is the default autoloader used in  `Module::getAutoloaderConfig()` implementations generated by the admin API. Since zend-loader is no longer a requirement of zend-mvc, we need to make it a requirement here going forward to ensure it's present.